static/css/card.css

```diff
@@ -0,0 +1,62 @@
+/* .card {
+    background-color: white;
+    border-radius: 0.2rem;
+    width: 18rem;
+    padding: 0.5rem;
+} */
+.card {
+    display: flex;
```

Comment:
try to organize css so its easier for you and others to navigate it. ie

selector {
  // related to background
  // related to width
  // related to positioning
  // related to text
  // related to border
}

group them. see mine `button.css` as example